ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -60,6 +60,7 @@ interface AddressSuggestion {
             [address]="inputAddress$ | async"
             [addressErrors]="addressErrors$ | async"
             [readonly]="isReadOnly$ | async"
+            [country]="staticCountry$ | async"
```

Comment:
We pass on the country to our child component. There is a little bit of logic for country beyond just having one set in config. More below.